vsts/determine_tests_to_run.ps1

```diff
@@ -13,11 +13,6 @@ function IsPullRequestBuild
 
 function ShouldSkipIotHubTests 
 {	
-	if (IsPullRequestBuild) 
```

Comment:
Were we not running tests earlier?

Reply:
Ahh draft PR, missed my eye :)

Reply:
we were actually doing the opposite, running tests irrespective of changes.